h/views.py

```diff
@@ -1,16 +1,18 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 import logging
-import re
 
 from pyramid import httpexceptions
 from pyramid.events import ContextFound
 from pyramid.view import forbidden_view_config, notfound_view_config
 from pyramid.view import view_config
+import pyramid.response
 
 from . import session
 from .models import Annotation
 from .resources import Application, Stream
+import h.api_client
```

Comment:
Ok, I'm happy to change all the import styles from `import h.foo` to `import .foo`, but are you sure we want to do `from . import ...`? The Python docs [warn against it](https://docs.python.org/2/howto/doanddont.html#from-module-import-name1-name2) because it duplicates name -> object bindings which can be a problem. It breaks `mock.patch`, for example.

I think we should stick to these styles:

    import .foo
    import .foo.bar
    import .foo.bar as foo
    import h.foo
    import .hfoo.bar
    import h.foo.bar as foo

and only use `from foo import ...` when really necessary.